requirements.txt

```diff
@@ -0,0 +1,496 @@
+absl-py==0.11.0
```

Comment:
Great, in fact necessary to add.
There seems to be a lot of libraries here. Are you positive these are all necessary?